android/AndroidManifest.xml

```diff
@@ -811,5 +811,9 @@
     <!-- Disable Google's anonymous stats collection -->
     <meta-data android:name="android.webkit.WebView.MetricsOptOut" android:value="true" />
 
+    <!-- For android Auto -->
+    <meta-data android:name="com.google.android.gms.car.application"
+        android:resource="@xml/automotive_app_desc"/>
```

Comment:
According to docs, you should change minSdkVersion from 21 to 23 in `android/gradle.properties`.

Reply:
Done